articles/windows-mdm-setup.md

```diff
@@ -2,30 +2,16 @@
 
 ![Windows MDM setup](../website/assets/images/articles/windows-mdm-fleet-1600x900@2x.png)
 
-## Overview
+To control OS settings, updates, and more on Windows hosts follow the manual enrollment instructions.
 
-Turning on Windows MDM features requires configuring Fleet with a certificate and key. This guide will walk you through how to upload these to Fleet and turn on Windows MDM.
+To use automatic enrollment (aka zero-touch) features on Windows, follow instructions to connect Fleet to Microsoft Azure Active Directory (aka Microsoft Entra). You can further customize zero-touch with Windows Autopilot.
 
-Automatic enrollment allows Windows workstations to automatically enroll to Fleet when they are first set up. Automatic enrollment requires Microsoft Azure Active Directory (aka Microsoft Entra). This guide will walk you through how to connect Azure AD to Fleet. 
+## Manual enrollment
 
-With Fleet connected to Azure AD, the end user will see Microsoft's default setup experience. You can further customize the initial setup with Windows Autopilot, which is similar to Apple's Automated Device Enrollment (DEP).
-
-## Requirements
-To use Fleet's Windows MDM features you need to have:
-- A [deployed Fleet instance](https://fleetdm.com/docs/deploy/introduction).
-- A Fleet user with the admin role.
-
-## Turning on Windows MDM
+### Step 1: generate your certificate and key
```

Comment:
I think awhile back we settled on the convention of capitalizing the word after "Step #:" (though isn't consistent everywhere yet)
```suggestion
### Step 1: Generate your certificate and key
```